src/Tribe/Views/V2/Views/Recent_Past_View.php

```diff
@@ -110,7 +140,10 @@ protected function setup_repository_args( Tribe__Context $context = null ) {
 		$this->repository = tribe_events();
 
 		$date                   = $context->get( 'event_date', 'now' );
-		$args['posts_per_page'] = $context->get( 'events_per_page', 3 );
+		//$args['posts_per_page'] = $context->get( 'events_per_page', 3 );
+
+		//todo mockup is to display 3 items, the context above uses a db setting. I think calling it recent_past_per_page with a default of 3 will fix this.
```

Comment:
:no_entry_sign: No space found before comment text; expected "// todo mockup is to display 3 items, the context above uses a db setting. I think calling it recent_past_per_page with a default of 3 will fix this." but found "//todo mockup is to display 3 items, the context above uses a db setting. I think calling it recent_past_per_page with a default of 3 will fix this."

<sub>hash: b5246439601211477173ce50791fe9d0</sub>

Reply:
I agree with this, it should be implemented now.
See my suggestions below.